src/components/LexiconClassViewer.tsx

```
@@ -808,6 +861,67 @@
+                                                  <button
+                                                    key={idx}
+                                                    className={`enum-value enum-value-deprecated ${copiedValue === value ? 'enum-value-copied' : ''}`}
+                                                    onClick={() => copyToClipboard(value)}
+                                                    title="Click to copy to clipboard"
+                                                  >
+                                                    {copiedValue === value ? (
+                                                      '✓ Copied!'
+                                                    ) : searchTerm && highlightedValue ? (
+                                                      <span
+                                                        dangerouslySetInnerHTML={{
+                                                          __html: highlightedValue,
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)